### PR TITLE
Fixes tessel erase

### LINKED ISSUE #313: t2 erase doesn't actually work
It does properly tell OpenWRT to stop running the service but it appears that OpenWRT still runs whatever is in `/app` at startup. This is exacerbated by the fact that `t2 erase` makes no effort to delete that folder.

### PR DESCRIPTION
**Pull request by johnnyman727**
Fixes #313.

- [x] Add tests

### CODE REVIEW

File: lib/tessel/commands.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ module.exports.scanWiFi = function() {
 module.exports.stopRunningScript = function() {
   return ['/etc/init.d/tessel-app', 'stop'];
 };
+module.exports.disablePushedScript = function() {
+  return ['/etc/init.d/tessel-app', 'disable'];
```
Comment (rwaldron) [inline lib/tessel/commands.js]: If the command is `erase`, is this misleading?

Comment (johnnyman727) [inline lib/tessel/commands.js]: @rwaldron the name is `erase` for legacy reasons. On Tessel 1, we literally just erased the code that was in flash to clear out a `tessel push`. I'm open to other suggestions. `tessel unpush`?

**Comment by johnnyman727** [comment]
@rwaldron could you give this one a review?

File: lib/tessel/commands.js (hunk 1 above)
Comment (rwaldron) [inline lib/tessel/commands.js]: We should think more on this and plan a follow up.

**Comment by rwaldron** [comment]
Works Great!

**Pull request merged**